lib/realtime/monitoring/prom_ex/plugins/phoenix.ex

```diff
@@ -57,15 +57,10 @@ if Code.ensure_loaded?(Phoenix) do
 
     def execute_metrics do
       active_conn =
-        case :ets.lookup(:ranch_server, {:listener_sup, HTTP}) do
-          [] ->
-            -1
-
-          _ ->
-            HTTP
-            |> :ranch_server.get_connections_sup()
-            |> :supervisor.count_children()
-            |> Keyword.get(:active)
+        if :ranch.info()[HTTP] do
```

Comment:
we don't test this change right? if we don't we should avoid changing this for now until we have tests around it as we already had issues with metrics in the past

Reply:
Good catch! I thought I had added a test! Will add one 👌

Reply:
Done here: https://github.com/supabase/realtime/pull/1378/commits/baaaf9a74b69a71c0c454f96702c20a348918947